lib/auth/password_test.go

```diff
@@ -507,15 +508,25 @@ func TestChangeUserAuthentication(t *testing.T) {
 				invalidReq := c.getInvalidReq(token.GetName())
 				_, err = srv.Auth().changeUserAuthentication(ctx, invalidReq)
 				require.True(t, trace.IsBadParameter(err))
+
+				if invalidReq.Passwordless {
+					require.Contains(t, err.Error(), "passwordless")
```

Comment:
Set a `wantErr` field in the test case instead of specializing for pwdless here?